src/main/java/seedu/letsgethired/model/application/CompanyContainsFieldKeywordsPredicate.java

```diff
@@ -65,8 +66,11 @@ public String getFieldValue(Prefix fieldPrefix, InternApplication internApplicat
         if (fieldPrefix.equals(PREFIX_CYCLE)) {
             return internApplication.getCycle().value;
         }
-        if (fieldPrefix.equals(PREFIX_NOTE)) {
-            return internApplication.getNote().value;
+        if (fieldPrefix.equals(PREFIX_NOTE_INSERT) || fieldPrefix.equals(PREFIX_NOTE_DELETE)) {
```

Comment:
Maybe just `PREFIX_NOTE_INSERT` here will be enough? I'm thinking we can assume `find` by the delete note prefix shouldn't be allowed, and it seems like the all the test cases for `find` just use `PREFIX_NOTE_INSERT` anyway.